[Whats'New][Restructuring][1705608] (#3121)
MicrosoftDocs/msteams-docs: msteams-platform/whats-new.md

>[!TIP]
> Check out sample solutions to common team scenarios in the [**Teams app template catalog**](samples/app-templates.md)! The catalog is alphabetized and the newest additions are tagged with a star ☆.

> The changes to the Microsoft Teams platform and this document set are listed in the following sections:

Reviewer: ```suggestion
Changes to the Microsoft Teams platform and this documentation are listed in the following sections:
```

> * [General Availability (GA) features](#ga-features): Provides a list of released features in Microsoft Teams. Some entries may be used to call attention to a new feature that is simply of interest to Teams developers.

Reviewer: ```suggestion
* [General availability (GA) features](#ga-features): Microsoft Teams platform features that are available to all app developers.
```
Slight edit since this list is only relevant to Teams app development and not end user or admin features.

> * [Developer preview](#developer-preview): Developer preview is a public program for developers, which provides early access to unreleased features in Microsoft Teams.  

Reviewer: ```suggestion
* [Developer preview](#developer-preview): Developer preview is a public program that provides early access to unreleased Teams platform features.  
```

|11/16/2020|Teams app manifest updated to version 1.8|[Reference: Manifest schema for Microsoft Teams](resources/schema/manifest-schema.md)|
|11/10/2020|Teams bot design guidelines|[Bot design guidelines](bots/design/bots.md)|
|9/30/2020|Sending and receiving files to bots on mobile devices is now supported.|[Send and receive files through your bot](resources/bot-v3/bots-files.md)|

> |09/22/2020|Get Started with Teams.|[Build your first Teams app overview](build-your-first-app/build-first-app-overview.md)|

Reviewer: ```suggestion
|09/22/2020|New information for getting started with Teams development.|[Build your first Teams app overview](build-your-first-app/build-first-app-overview.md)|
```

| 02/05/2018 | Added topics for getting started using C#. |[Get started on the Microsoft Teams platform with C#/.NET](./get-started/get-started-dotnet-app-studio.md)|

## Developer preview

| **Date** | **Notes** | **Changed topics** |
| -------- | --------- | ------------------ |
|04/08/2021| App customization|[Design teams app overview](concepts/design/design-teams-app-overview.md#app-customization), [App studio overview](concepts/build-and-test/app-studio-overview.md#connectors), and [Manifest schema](resources/schema/manifest-schema-dev-preview.md) |
|03/05/2021| Tabs will no longer have margins surrounding their experiences. Tab developers should review and update their apps. | [Removing tab margins](resources/removing-tab-margins.md) |
|03/05/2021 | Default install scope and group capability.| [Default install scope and group capability](concepts/deploy-and-publish/apps-upload.md#add-a-default-install-scope-and-group-capability) |
|03/04/2021|Information masking in Adaptive cards.| [Information masking in Adaptive cards](task-modules-and-cards/cards/cards-format.md#information-masking-in-adaptive-cards) |

> |9/18/2020|Support for in-meeting Teams apps (Release Preview)|[Create apps for Teams meetings](apps-in-teams-meetings/create-apps-for-teams-meetings.md) and [Apps in Teams meetings](apps-in-teams-meetings/teams-apps-in-meetings.md)|

Reviewer 2: I think anything before 2021 should be in R4 already. Can we please confirm with respective feature PM?

Author: Ok. Will confirm
